core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheConfig.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.tieredstorage.chunkmanager.cache;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+public class ChunkCacheConfig extends AbstractConfig {
+    private static final String CACHE_SIZE_CONFIG = "size";
+    private static final String CACHE_SIZE_DOC = "Cache size in bytes";
+    private static final String CACHE_RETENTION_CONFIG = "retention.ms";
+    private static final String CACHE_RETENTION_DOC = "Cache retention time ms";
+    private static final long DEFAULT_CACHE_SIZE_BYTES = 100 * 1024 * 1024;
+    private static final long DEFAULT_CACHE_RETENTION_MS = 600_000;
+
+    private static ConfigDef addCacheConfigs(final ConfigDef configDef) {
+        configDef.define(
+                CACHE_SIZE_CONFIG,
+                ConfigDef.Type.LONG,
+                DEFAULT_CACHE_SIZE_BYTES,
```

Comment:
Maybe let's not introduce the default value? Sounds important enough to make users make explicit decision

Reply:
Fixed